doodle.py

```diff
@@ -65,13 +65,51 @@ def main():
 
 # Parse date string and return date and whether it was relative or not
 def derelativize_date(datestr: str, base=datetime.now()):
-    if datestr == '':
-        return base.replace(hour=0, minute=0, second=0, microsecond=0), True
+    relative = False
 
     if not datestr.startswith('+'):
-        return datetime.strptime(datestr, '%Y-%m-%d')
+        return datetime.strptime(datestr, '%Y-%m-%d'), relative
 
-    return base.replace(hour=0, minute=0, second=0, microsecond=0) + timedelta(days=int(datestr[1:])), True
+    else:
+        relative = True
+        date = base.replace(hour=0, minute=0, second=0, microsecond=0)
+        try:
+            # Try to add date
+            date = date + timedelta(days=int(datestr[1:]))
+        except ValueError:
+            # Leave the date as it is.
+            date = date
```

Comment:
Erroring silently for this is likely a bad idea. Wrong date formats should error earlier to prevent creating unwanted polls.